CONTRIBUTING.rst

```diff
@@ -1,134 +1,32 @@
-Development Setup
-=================
+================
+Developing MetaCI via Docker
+================
 
-Cloning the project
--------------------
+MetaCI can be setup using either docker virtual containers
+or on your local machine. To set up MetaCI using docker please
+see the following instructions `RUNNING_DOCKER.rst`_.
```

Comment:
```suggestion
see the following instructions `<./RUNNING_DOCKER.rst>`_.
```

Then it works as a relative link and you don't need the line starting with `.. _RUNNING_DOCKER.rst` below.

(And I'm asking for this change for all the links that are currently using github.com URLs)